about.html

```diff
@@ -0,0 +1,145 @@
+            </ul>
+            <a href="#nav-bar" class="hamburger" aria-label="Open main menu"><i class="fas fa-bars" aria-hiiden="true"></i></a>
+        </nav>
+    </header>
+
+    <main class="main-section">
+        <div class="main-section-row">
+            <div class="about-container">
+                <h1>
+                    Cyberworld Security <br />
+                    Global Summit
+                    2015
+                </h1>
+                <hr />
+                <p>
+                    The Global Summit brings together the community of experts, academics, and activists who comprise the Cyber Security affiliate network in a different country every two years.
+
+                    This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that that advance the cause of the Cyberworld, free culture and open knowledge.
+
+                    So if you're active and engaged in the worlds of open content and knowledge - free software advocates, Wikipedians, Open Knowledge, galleris, libraries, museums, archives, governments and foundations, lawyers and activists - we hope you'll consider joining us this year to build a stronger, more vibrant cyberworld together 
+                </p>
+                
+                <h2>
+                    Please contact us per Email for any further questions about CyberWorld Global Summit 2105!
+                    <br /><u>cyberworldsummit2015@cyberworldkorea.org</u>
+                </h2>
+            </div>
+        </div>
+    </main>
+
+    <section class="summit-section">
+        <div class="summit-section-row">
+
+            <div class="summit-container">
+                <h2>Creative Commons Global Summit 2015 logo</h2>
+                <hr />
+                <p>
+                    The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+                    After online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we optimized the prize-winning-logo for this year's summit as following:
+                </p>
+                <div>
+                    <img src="img/cc-summit.png" alt="" />
+                </div>
+               
+            </div>
+
+        </div>
+    </section>
+
+    <section class="past-summit-section">
+
+        <div class="past-summit-section-row1">
+
+            <div>
+                <h2>See the past CC Global Summit</h2>
+                <hr />
+               <p>Take a look at the past CC Global Summit which took place in Beunos Aires and Warsaw</p>
+            </div>
+
+        </div>
+
+        <div class="past-summit-section-row2">
+
+            <div class="div1">
+                <h2 class="timeline">2013</h2>
+                <p>CC Global Summit 2013 in Buenos Aires</p>
+            </div>
+

```

Comment:
- [ ] Please fix the positioning and width of the `Summit section`, `about logo`, and `past summits` sections because now those sections are overflowing in the mobile view.

![chrome-capture-2022-5-23 (1)](https://user-images.githubusercontent.com/68796232/175383720-7821169e-55e9-4f44-b57b-1fda4051dbe9.gif)

Reply:
I have fixed the positioning as requested